partcad/src/partcad/assembly_factory.py

```
@@ -23,16 +23,17 @@ def __init__(self, ctx, source_project, target_project, config, extension=""):
         self.name = config["name"]
         self.orig_name = config["orig_name"]
 
-    def _create(self, config):
-        self.assembly = Assembly(config)
-        self.assembly.project_name = self.project.name  # TODO(clairbee): pass it via the constructor
-        # TODO(clairbee): Make the next line work for assembly_factory_file only
-        if self.path:
-            self.assembly.path = self.path
-        self.project.assemblies[self.name] = self.assembly
-
+    def _create(self, config) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `config`

(ANN001)